modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -933,6 +933,34 @@ public void testAddObjectEntry() throws Exception {
 					).build()
 				).build()));
 
+		Map<String, Serializable> richTextVideo =
+			HashMapBuilder.<String, Serializable>put(
+				"richTextLocalized_i18n",
+				HashMapBuilder.put(
+					"en_US",
+					StringBundler.concat(
+						"<div class=\"embed-responsive ",
+						"embed-responsive-16by9\" data-embed-id=",
+						"\"https://www.youtube.com/embed/6LjQ7Z99N74?rel=0\" ",
+						"data-styles=\"{&quot;width&quot;:&quot;81%&quot;}",
+						"\" style=\"width:81%\"><iframe allow=\"autoplay; ",
+						"encrypted-media\" allowfullscreen=\"\" ",
+						"frameborder=\"0\" height=\"315\" src=",
+						"\"https://www.youtube.com/embed/6LjQ7Z99N74?rel=0\" ",
+						"width=\"560\"></iframe></div> <p>&nbsp;</p>")
+				).build()
+			).build();
+
+		objectEntry = _addObjectEntry(
+			group.getGroupId(), objectDefinition.getObjectDefinitionId(),
+			richTextVideo);
+
+		values = objectEntry.getValues();
+
+		Assert.assertEquals(
+			richTextVideo.get("richTextLocalized_i18n"),
+			values.get("richTextLocalized_i18n"));
```

Comment:
@G-Sa I'd create another test for this use case because I believe it's to specific for this test, I'd use the same pattern you use in the CalendarBooking test.

Rename `richTextVideo` to `expectedValues` and define : 

`Map<String, Serializable> actualValues = objectEntry.getValues();`

you can also create an rich text object field that is not localized to make the test easier to understand.

Thanks!